### DISCUSSION TITLE
wire navigate  shows cache on full page component

### DISCUSSION THREAD

**Discussion by aftabkhaliq86** [livewire/livewire#7597] [1. Help]
using wire navigate   when click on link  page navigated to next page but sometimes it just show HTML cached page but when click on the same link again fetch the new updated HTML with content

**Comment by joshhanley** [comment]
We're in the process of investigating all wire:navigate issues and we can't replicate this with the information give so I'm going to close this. If you're still experiencing problems, please open a new discussion and provide code examples that demonstrate the issue that we can copy and paste so we can investigate. Thanks!
